.babelrc

```diff
@@ -5,15 +5,20 @@
   "env": {
     "cjs": {
       "presets": [
-        ["es2015", {"loose": true}]
+        ["latest", {"es2015": {"loose": true}}]
```

Comment:
Admittedly yes :) I can get rid of that and use the traditional promise approach, or we could use babel-preset-env as discussed below in lieu of babel-preset-2015 / babel-preset-next, and use it to manage all the builds. which could be good